relay-event-schema/src/protocol/replay.rs

```diff
@@ -82,6 +82,7 @@ pub struct Replay {
     ///   "replay_type": "session"
     /// }
     /// ```
+    #[metastructure(bag_size = "small")]
     pub replay_type: Annotated<String>,
```

Comment:
For string fields, we have an attribute called `max_chars` which might be better suited here, see for example https://github.com/getsentry/relay/blob/5de78d9b5e89a42e27819b601867c9026dc57a93/relay-event-schema/src/protocol/event.rs#L297-L298.

Limits for each category are listed here:

https://github.com/getsentry/relay/blob/5de78d9b5e89a42e27819b601867c9026dc57a93/relay-event-schema/src/processor/attrs.rs#L116-L139